app/AttributeTypes/SiUnitAttribute.php

```diff
@@ -2,338 +2,27 @@
-                        'label' => 'gram',
-                    ],
-                    [
-                        'symbol' => 'kg',
-                        'convert' => fn($v) => $v,
-                        'label' => 'kilogram',
-                    ],
-                    [
-                        'symbol' => 't',
-                        'convert' => fn($v) => $v * pow(10, 3),
-                        'label' => 'ton',
-                    ],
-                    [
-                        'symbol' => 'oz',
-                        'convert' => fn($v) => $v * 0.028349523125,
-                        'label' => 'ounce',
-                    ],
-                    [
-                        'symbol' => 'lb',
-                        'convert' => fn($v) => $v * 0.45359237,
-                        'label' => 'pound',
-                    ],
-                ],
-            ],
-            'temperature' => [
-                'default' => 'degree_celsius',
-                'units' => [
-                    [
-                        'symbol' => '°C',
-                        'convert' => fn($v) => $v,
-                        'label' => 'degree_celsius',
-                    ],
-                    [
-                        'symbol' => '°K',
-                        'convert' => fn($v) => $v - 273,15,
-                        'label' => 'degree_kelvin',
-                    ],
-                    [
-                        'symbol' => '°F',
-                        'convert' => fn($v) => ($v - 32) / 1,8,
-                        'label' => 'degree_fahrenheit',
-                    ],
-                    [
-                        'symbol' => '°R',
-                        'convert' => fn($v) => $v * 1.25,
-                        'label' => 'degree_réaumur',
-                    ],
-                ],
-            ],
-            // 'substance' => [],
-            // 'luminous' => [],
-            // 'energy' => [],
-            // 'discspace' => [
-            //     // b, kb, mb, gb, tb, pb
-            //     // bi, kibi, mibi, gibi, tibi, pibi
-            // ],
-        ];
-    }
-
-    private static function getUnitFromKey(string $value, string $key) : array|null {
-        if(!isset($key) || !isset($value)) return null;
 
-        foreach(self::getUnits() as $grpLabel => $grp) {
-            foreach($grp['units'] as $unit) {
-                if(is_array($unit[$key])) {
-                    if(in_array($value, $unit[$key])) {
-                        return $unit;
-                    }
-                } else {
-                    if($unit[$key] == $value) {
-                        return $unit;
-                    }
-                }
-            }
+    public static function getGlobalData() : array {
+        $unitsArray = [];
+        foreach(UnitManager::get()->getUnitSystems() as $unitSystem) {
+            $unitsArray[$unitSystem->name] = $unitSystem->toArray();
```

Comment:
Renamed from 'addGlobalData' as we just retrieve the unit system array. 
Not adding anything.